src/Umbraco.Web/Editors/ContentTypeController.cs

```diff
@@ -322,6 +322,23 @@ public DocumentTypeDisplay PostSave(DocumentTypeSave contentTypeSave)
             return display;
         }
 
+        public TemplateDisplay PostCreateDefaultTemplate(int id)
+        {
+            var contentType = Services.ContentTypeService.Get(id);
+            if (contentType == null)
+            {
+                throw new NullReferenceException("No content type found with id " + id);
+            }
+
+            var template = CreateTemplateForContentType(contentType.Alias, contentType.Name);
+            if (template == null)
+            {
+                throw new NullReferenceException("Could not create default template for content type with id " + id);
```

Comment:
We don't really want to ever throw a NullReferenceException, that exception should be reserved by .NET itself and only used to indicate that you have a null object reference.  In this case it should be InvalidOperationException, see https://docs.microsoft.com/en-us/dotnet/standard/exceptions/best-practices-for-exceptions

Reply:
Gotcha. I was just coding for consistency, given that NullReferenceException is used elsewhere in `ContentController` - but when I look closer it also uses `HttpResponseException` so my bad 👍 

Both exceptions are fixed in 1aec0e1